frontend/server/src/DAO/TeamUsers.php

```diff
@@ -34,4 +34,24 @@ public static function getByTeamId(int $teamId): array {
         }
         return $usersTeams;
     }
+
+    /**
+     * @param list<string> $usernames
+     */
+    public static function createTeamUsersBulk(
+        int $teamId,
+        array $usernames
+    ): int {
+        $placeholders = array_fill(0, count($usernames), '?');
+        $placeholders = join(',', $placeholders);
+        $sql = "REPLACE INTO Team_Users (team_id, user_id)
+                SELECT {$teamId} AS team_id, user_id FROM Identities
```

Comment:
```suggestion
                SELECT ? AS team_id, user_id FROM Identities
```

en general nunca hay que embeber variables en una consulta. si el ímpetu de esto es evitar manipular el segundo parámetro de `Execute()` (que ahorita es `$usernames`), para resolver este problema hay que evitar _evitar manipularlo_:

```php
        \OmegaUp\MySQLConnection::getInstance()->Execute(
            $sql,
            array_merge([$teamId], $usernames)
        );
```

Reply:
Listo. Ya quedó.